apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -288,7 +288,7 @@ describe('Notifications Creator', function () {
       cy.wait(1000);
       editChannel('inApp');
 
-      cy.getByTestId('feed-button-0').should('be.checked');
+      cy.getByTestId('feed-button-0-checked');
```

Comment:
Do you think we better add .should("be.visible") to make sure it is?

Reply:
the thing here is that I do not test if it is visible or not but if it is the selected one...